File: app/Models/inscription.py
class Inscription():
    student_id: str
    subject_code: str

    def __init__(self, student_id, subject_code):
        self.student_id = student_id
        self.subject_code = subject_code

    def toJson(self):
        return {
            "student_id": self.student_id,
            "subject_code": self.subject_code
        }

File: src/Models/inscription.js
'use strict'

class Inscription {
    static SUBJECT_LIST = {
        "AM1":"Analisis Matematico I",
        "AM2": "Analisis Matematico II",
        "F1": "Fisica I",
        "ALGO1": "Algoritmos y programacion I"
    }

    constructor(student_id, subject_code){
        this.student_id = student_id;
        this.subject_code = subject_code;
    }

    static exists(subject_code){
        return subject_code in this.SUBJECT_LIST; 
    }
}

module.exports = Inscription;